src/SWP/Bundle/ContentBundle/EventListener/ProcessArticleMediaListener.php

```diff
@@ -166,6 +175,42 @@ public function createImageMedia($articleMedia, $item)
     }
 
     /**
+     * @param ArticleInterface $article
+     * @param ArticleMedia     $articleMedia
+     */
+    private function replaceBodyImagesWithMedia($article, $articleMedia)
```

Comment:
this can be moved to a separate class and injected here, this listener is really complex atm and its getting more complex with every PR, we should consider to improve it with the next iterations maybe, not now